src/components/flowchart-wrapper/flowchart-wrapper.js

```diff
@@ -330,7 +331,7 @@ export const FlowChartWrapper = ({
               'pipeline-wrapper__go-back-btn--show':
                 goBackToExperimentTracking?.showGoBackBtn,
               'pipeline-wrapper__go-back-btn--show-sidebar-visible':
-                sidebarVisible,
+                sidebarVisible || displaySidebar,
```

Comment:
why do we need both displaySidebar and sidebarVisible ? I think I do not understand fully on what it means when we say -
sidebarVisible = true/false and 
displaySidebar = true/false 

Could you please explain @jitu5 . Thank you

Reply:
I see we have sidebar in both visible and display. If that is the case then shouldn't the class `pipeline-wrapper__go-back-btn--show-sidebar-visible` only appear when sidebarVisible is true ? -
```
     display: {
        sidebar: true,
        expandAllPipelines: false,
         ...
      }
      visible: {
        ...
        pipelineBtn: true,
        sidebar: true,        
      }
```

Reply:
@ravi-kumar-pilla 
displaySidebar removes sidebar completely where sidebarVisible just collapse sidebar. but correct we dont need displaySidebar for pipeline-wrapper__go-back-btn--show-sidebar-visible, so I removed it.